### ISSUE TITLE
Trending Collection

### ISSUE THREAD

**Issue by gitjeet** [WalkUpLabs/MapX_Artist_FE#4]

**Comment by gitjeet** [comment]
@pathaksoham2003 could add the end date estimate on all ticket and could you update the status of the ticket

**Comment by pathaksoham2003** [comment]
Yes sure i will update them i will be travelling to mumbai for 2 weeks
onsite for my company but will surely complete things on time and update
them accordingly!
